driverapi/driverapi.go

```diff
@@ -74,6 +74,9 @@ type Driver interface {
 
 	// Type returns the the type of this driver, the network type this driver manages
 	Type() string
+
+	// IsBuiltIn returns true if it is a built-in driver
+	IsBuiltIn() bool
```

Comment:
Hmmm, it looks like you need this info outside of `BuiltInDrivers()`. Then your current change is probably what is needed.